aws/table_aws_backup_legal_hold.go

```diff
@@ -0,0 +1,187 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/backup"
+	"github.com/aws/aws-sdk-go-v2/service/backup/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBackupLegalHold(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_backup_legal_hold",
+		Description: "AWS Backup LegalHold",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("legal_hold_id"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValueException"}),
+			},
+			Hydrate: getAwsBackupLegalHold,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBackupLegalHolds,
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "legal_hold_id",
+				Description: "ID of specific legal hold on one or more recovery points.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "This is an Amazon Resource Number (ARN) that uniquely identifies the legal hold.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("LegalHoldArn"),
+			},
+			{
+				Name:        "creation_date",
+				Description: "This is the time in number format when legal hold was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "status",
+				Description: "This is the status of the legal hold. Statuses can be ACTIVE, CREATING, CANCELED, and CANCELING.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "cancellation_date",
+				Description: "This is the time in number format when legal hold was cancelled.",
+				Type:        proto.ColumnType_TIMESTAMP,
```

Comment:
What does the data look like for this column? Does it come as epoch time, or something else? If so, we should convert to the standard date/time format we use for other columns.

Reply:
In Go SDK, we are getting the value of `CancellationData` of type `*time.Time`, it is not coming as an epoch time